### ISSUE TITLE
[F-Req] Support choosing 2FA method

### ISSUE THREAD

**Issue by Zhou-Shilin** [0xkuj/IPARanger#22]
My Apple ID account linked to 2 phone numbers and the previous one I didn't use anymore. In App Store, I can choose "Cannot receive the message -> Send to ******(my phone number)" but there isn't this option in iparange.
Sorry for my bad English.

**Comment by 0xkuj** [comment]
you are right, it is not an option. you need to have a device that is connected to the account that asks the 2fa in order to get the 2fa notification. sorry about that, i cant control that

**Comment by Zhou-Shilin** [comment]
> you are right, it is not an option. you need to have a device that is connected to the account that asks the 2fa in order to get the 2fa notification. sorry about that, i cant control that

Thanks for your reply. I do have a device that is connected to the account but it's not apple's first choice. Apple always send the verification code to my old phone first, and I need to choose the second phone that connected to my account.

**Comment by 0xkuj** [comment]
unfortunately i cant control that, this is what the API supplies.
can you change your main device so it wont be your old device that gets the 2fa?

**Comment by Zhou-Shilin** [comment]
> unfortunately i cant control that, this is what the API supplies.
> can you change your main device so it wont be your old device that gets the 2fa?

i can't. because it's not my main apple id and i use it for downloading apps which is only available for another country, i cannot login the account on an idevice except the appstore.

**Comment by 0xkuj** [comment]
Sorry, this won't be fixed as this is what Apple API provides
